Nitrox.Assets.Subnautica/LanguageFiles/en.json

```diff
@@ -75,6 +75,7 @@
     "Nitrox_ResyncSettings": "Resynchronization",
     "Nitrox_SafeBuilding": "Safe building",
     "Nitrox_SafeBuildingLog": "Safe building log",
+    "Nitrox_ScannerRoomWarn": "Please note: scanner rooms do not work properly in this version of Nitrox. Expect many bugs.",
```

Comment:
Maybe we could create a more generic one like "This command is not available in Nitrox", so we can reuse that one in area we're not okay with the current sync

Reply:
That would make it unclear what was causing the issue though. Maybe it could be a generic message with a place to insert what we are referring to: "This feature is not yet implemented in Nitrox, expect many bugs: Scanner Room"